rosbag2_transport/test/rosbag2_transport/test_play_seek.cpp

```diff
@@ -103,7 +103,7 @@ class RosBag2PlaySeekTestFixture
 };
 
 TEST_P(RosBag2PlaySeekTestFixture, seek_back_in_time) {
-  const size_t expected_number_of_messages = num_msgs_in_bag_ + 2;
+  const size_t expected_number_of_messages = num_msgs_in_bag_ + 4;
```

Comment:
We expect all remaining messages to be played after resuming, so this should be 4.